packages/salesforcedx-vscode-core/README.md

```diff
@@ -60,8 +61,7 @@ To check your running tasks, expand the Running Tasks view in the Explorer.
 ![Running Tasks view, showing that Apex tests are running](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/running_tasks.png)
 
 ## Run Apex Tests
-To run Apex tests, in your `.cls` file, click the name of an Apex test class or method. Next, click the code actions lightbulb that appears by the class or method signature, and then select **SFDX: Invoke Apex Test Method** or **SFDX: Invoke All Apex Tests in Class**.  
-![GIF that shows running Apex tests for a method and then a class](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-core/images/apex_test_run.gif)  
```

Comment:
We should delete this gif or replace it with a new one that showcases the new behavior.

Reply:
So, @ruthemmanuelle is removing the reference to the gif for this release for now. She will put a new one in next week when she has time. I agree with what she is doing. I think it's OK to have an old apex_test_run.gif that is not referenced yet. She can update the file (override it) next time.